configuring.tex

```diff
@@ -97,6 +97,27 @@ \section{Preparing for First Use: Formatting SD Cards}
 Record (``Boot Sector'') by instead typing ``FIX MBR,'' should the
 need arise.
 
+
+As an end result, we want to have a correctly formatted SD card with the essential files stored on it for MEGA65 to boot from.
+
+This is how it works: When powering on, MEGA65 will search for- and boot these files:
+\begin{itemize}
+\item {\tt FREEZER.M65}
+\item {\tt AUDIOMIX.M65}
+\item {\tt C64THUMB.M65
+\item {\tt C65THUMB.M65
+\item {\tt MEGA65.ROM
+\item {\tt MEGA65.D81 (default disk image, automatically mounted during start)
```

Comment:
Close the {\tt ...} after the file name, so that the comment after will be rendered in the normal font.